Client/Frontend/Browser/Tab.swift

```diff
@@ -276,6 +278,14 @@ class Tab: NSObject {
     rewardsId = UInt32.random(in: 1...UInt32.max)
     nightMode = Preferences.General.nightModeEnabled.value
     syncTab = tabGeneratorAPI?.createBraveSyncTab(isOffTheRecord: type == .private)
+    
+    if let syncTab = syncTab {
+      faviconDriver = FaviconDriver(webState: syncTab.webState).then {
+        $0.setMaximumFaviconImageSize(1024)
```

Comment:
Do we need to set the maximum image size everytime we set a driver ?

Reply:
Yeah we need to. I made it this way on purpose. All it does is set an integer in Brave-Core, so that any favicons downloaded will be restricted by that size. This is because some Favicons are absolutely HUGE, and we need this number so we can scale them down if they're larger.